documentation/api/v2_0.rst

```diff
@@ -6,15 +6,27 @@ Version 2.0
 Summary
 -------
 
+.. qrefflask:: flexmeasures.app:create(env="documentation")
+    :modules: flexmeasures.api.v2_0.implementations.sensor_data
+    :order: path
+    :include-empty-docstring:
+
 .. qrefflask:: flexmeasures.app:create(env="documentation")
     :blueprints: flexmeasures_api, flexmeasures_api_play, flexmeasures_api_v2_0
     :order: path
     :include-empty-docstring:
+    :undoc-endpoints: flexmeasures_api_v2_0.get_meter_data, flexmeasures_api_v2_0.get_prognosis, flexmeasures_api_v2_0.post_meter_data, flexmeasures_api_v2_0.post_price_data, flexmeasures_api_v2_0.post_prognosis, flexmeasures_api_v2_0.post_weather_data
```

Comment:
Is that work in progress because these endpoints are undocumented, but still available?

Anyway, wouldn't it maybe be even cleaner to start fresh with v3.0? This PR could simply start v3.0 and leave v2.0 in the state it is now, to be deprecated in the big purge soon.